lib/zlib.js

```diff
@@ -453,7 +453,9 @@ Zlib.prototype.flush = function(kind, callback) {
   }
 };
 
-Zlib.prototype.close = function(callback) {
+Zlib.prototype.close = function(callback, options) {
+  options = options || {};
+
   if (callback)
```

Comment:
I don't think that will work unless we're OK breaking any existing userland code that looks like this:

```
'use strict';
const zlib = require('zlib');

const zipper = new zlib.Gzip();
zipper.on('close', () => { // do something });
zipper.close();
```

No idea how common that is. It's certainly atypical, as `close()` is undocumented, but it is exposed...